### PR TITLE
Feature/configurable resources

### PR DESCRIPTION
**Pull request by nicklloyd**
- [x] display configurable resources on connection view
 - [x] link to configuration form that loads connection configuration defaults per resource
 - [x] push form values back to unify
 - [x] extend simple inputs to respond to type (and validation coming)
 - [x] add proper error handling
 - [ ] ...

### CODE REVIEW

**Comment by Gdewilde** [comment]
#### 📝Changed routes:


| [**…nified-api\]/\[provider\]**](https://integration-settings-a9fvkhb38-apideck.vercel.app/integrations/[unified-api]/[provider]) | [**…/\[provider\]/\[resource\]**](https://integration-settings-a9fvkhb38-apideck.vercel.app/integrations/[unified-api]/[provider]/[resource]) |
|:-:|:-:|
|<a href="https://integration-settings-a9fvkhb38-apideck.vercel.app/integrations/[unified-api]/[provider]">Screenshot of /integrations/[unified-api]/[provider]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/p9nDwIQB3VEQZS2CnoCuJ7Skjg8_.png">(view full size)</a> |<a href="https://integration-settings-a9fvkhb38-apideck.vercel.app/integrations/[unified-api]/[provider]/[resource]">Screenshot of /integrations/[unified-api]/[provider]/[resource]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/NHL3qyO7pX3qNc7vywugjsgzabA_.png">(view full size)</a> |







Commit ce1a7a72d07b988f08b72c88c4d12a59792bbcd6 (https://integration-settings-a9fvkhb38-apideck.vercel.app).

File: __tests__/unit/components/Inputs/TextInput.test.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      name: 'CheckBox',
+      type: 'checkbox',
+      required: true,
+      placeholder: 'Test Placeholder',
+      onChange: () => {},
+      onBlur: () => {}
+    }
+
+    it('should render the component', async () => {
+      render(<TextInput {...props} />)
```
Comment (nicklloyd) [inline __tests__/unit/components/Inputs/TextInput.test.tsx:77]: is a dedicated CheckBox component now no?

Comment (jakeprins) [reply __tests__/unit/components/Inputs/TextInput.test.tsx:77]: oh, yea I changed that in one of the last commits. Good catch!

**Review by jakeprins — APPROVED** [review]

**Comment by Gdewilde** [comment]
Great job guys! 🙌🏻

**Pull request merged**